src/metal.cs

```diff
@@ -3614,109 +4094,417 @@ partial interface MTLRenderCommandEncoder : MTLCommandEncoder {
 
 #if NET
+#if !TVOS
+		[Mac (12, 0), iOS (15, 0), NoTV, MacCatalyst (15, 0), NoWatch]
 		[Abstract]
+#else
+		[NoMac, NoiOS, TV (16,0), NoMacCatalyst, NoWatch]
+#endif
+
+#else
+		[Mac (12, 0), iOS (15, 0), TV (16, 0), MacCatalyst (15, 0), NoWatch]
 #endif
-		[Mac (12, 0), iOS (15, 0), NoTV, MacCatalyst (15, 0), NoWatch]
 		[Export ("setTileVisibleFunctionTables:withBufferRange:")]
 		void SetTileVisibleFunctionTables (IMTLVisibleFunctionTable [] functionTables, NSRange range);
+
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), NoWatch]
+#if NET
+		[Abstract]
+#endif
+		[Export ("setVertexBuffer:offset:attributeStride:atIndex:")]
+		void SetVertexBuffer ([NullAllowed] IMTLBuffer buffer, nuint offset, nuint stride, nuint index);
+
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), NoWatch]
+#if NET
+		[Abstract]
+#endif
+		[Export ("setVertexBuffers:offsets:attributeStrides:withRange:")]
+		void SetVertexBuffers (IMTLBuffer [] buffers, IntPtr offsets, IntPtr strides, NSRange range);
+
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), NoWatch]
+#if NET
+		[Abstract]
+#endif
+		[Export ("setVertexBufferOffset:attributeStride:atIndex:")]
+		void SetVertexBufferOffset (nuint offset, nuint stride, nuint index);
+
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), NoWatch]
+#if NET
+		[Abstract]
+#endif
+		[Export ("setVertexBytes:length:attributeStride:atIndex:")]
+		void SetVertexBytes (IntPtr bytes, nuint length, nuint stride, nuint index);
+
+		[Mac (13, 0), iOS (16, 0), TV (16, 0), MacCatalyst (16, 0)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
+		[Export ("drawMeshThreadgroups:threadsPerObjectThreadgroup:threadsPerMeshThreadgroup:")]
+		void DrawMeshThreadgroups (MTLSize threadgroupsPerGrid, MTLSize threadsPerObjectThreadgroup, MTLSize threadsPerMeshThreadgroup);
+
+		[Mac (13, 0), iOS (16, 0), TV (16, 0), MacCatalyst (16, 0)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
+		[Export ("drawMeshThreadgroupsWithIndirectBuffer:indirectBufferOffset:threadsPerObjectThreadgroup:threadsPerMeshThreadgroup:")]
+		void DrawMeshThreadgroups (IMTLBuffer indirectBuffer, nuint indirectBufferOffset, MTLSize threadsPerObjectThreadgroup, MTLSize threadsPerMeshThreadgroup);
+
+		[Mac (13, 0), iOS (16, 0), TV (16, 0), MacCatalyst (16, 0)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
+		[Export ("drawMeshThreads:threadsPerObjectThreadgroup:threadsPerMeshThreadgroup:")]
+		void DrawMeshThreads (MTLSize threadsPerGrid, MTLSize threadsPerObjectThreadgroup, MTLSize threadsPerMeshThreadgroup);
+
+		[Mac (13, 0), iOS (16, 0), TV (16, 0), MacCatalyst (16, 0)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
+		[Export ("setMeshBufferOffset:atIndex:")]
+		void SetMeshBufferOffset (nuint offset, nuint index);
+
+		[Mac (13, 0), iOS (16, 0), TV (16, 0), MacCatalyst (16, 0)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
+		[Export ("setMeshBuffers:offsets:withRange:")]
+		void SetMeshBuffers (IMTLBuffer [] buffers, IntPtr offsets, NSRange range);
```

Comment:
Same, C-array: https://developer.apple.com/documentation/metal/mtlrendercommandencoder/3929922-setmeshbuffers?changes=latest_major

```suggestion
		void SetMeshBuffers (IntPtr buffers, IntPtr offsets, NSRange range);
```